ReactNativeClient/lib/models/Tag.js

```diff
@@ -174,14 +210,121 @@ class Tag extends BaseItem {
 		}
 	}
 
+	static tagPath(tags, tagId) {
+		const idToTags = {};
+		for (let i = 0; i < tags.length; i++) {
+			idToTags[tags[i].id] = tags[i];
+		}
+
+		const path = [];
+		while (tagId) {
+			const tag = idToTags[tagId];
+			if (!tag) break; // Shouldn't happen
+			path.push(tag);
+			tagId = tag.parent_id;
+		}
+
+		path.reverse();
+
+		return path;
+	}
```

Comment:
Please create a utility method shared by Folder.folderPath and tag.tagPath

Reply:
I've created a `nested-utils.js` file, since I wasn't sure where else that utility could go.